core/pool.go

```diff
@@ -9,7 +9,7 @@ import (
 type Pool struct {
 }
 
-func (p Pool) Start(engines []Engine, provider RequestProvider, metricsManager MetricsManager, config ChukonuConfig) {
+func (p Pool) Start(engines []Engine, provider RequestProvider, metricsManager MetricsManager, config ChukonuConfig, fuse chan bool, ack chan bool) {
```

Comment:
So you are right, we do need a way, in addition to the fuse, to dynamically adjust the workers after all workers start working(stablized).